build/pipelines/azure-pipelines.release.yaml

```diff
@@ -16,8 +16,8 @@ pr: none
 
 variables:
   versionMajor: 10
-  versionMinor: 2103
-  versionBuild: $[counter('10.2103.*', 0)]
+  versionMinor: 2108
```

Comment:
Hi @mcooley, is it okay to bump the version to 2018 even though we're going to release it before August?

Reply:
This is OK.